### DISCUSSION TITLE
Dağıtılan İbax Miktarı !!!!

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#798] [Q&A]
Arzına göre Çok ama çok az Ödül dağıtıyorlar. Zaten bunların bi kısmı Youtuber lara gidicek biz gene bişey kalmıyacak...
Lütfen artırılsın...
Beyler yeterince Seslenirsek bence artırırlar En azından bi deneyelim bir el atın.

![samsung_galaxy_fold_blue_butterfly_4k-1600x900](https://user-images.githubusercontent.com/98268472/151061292-9d5e9985-20fe-451f-baf5-9b727d7b6785.jpg)

**Comment by ghost** [comment]
O konuda haklısın umarım artırırlar :( :(

**Comment by cryptopus58** [comment]
+57647955
